src/img/icons.json

```diff
@@ -25,6 +25,7 @@
     "circle-eye",
     "circle-person",
     "circle-plus",
+    "circle-plus-2",
```

Comment:
These two are exactly the same, can we use the existing one and use CSS to change the color to blue.